internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -146,22 +147,25 @@ func resourceKustoCluster() *pluginsdk.Resource {
 				MaxItems: 1,
 				Elem: &pluginsdk.Resource{
 					Schema: map[string]*pluginsdk.Schema{
+						"state": {
```

Comment:
I missed the context that this block is deprecated so the orginal implementation of this was correct that we can Enable/Disable this for the user. Apologies for missing that context

Reply:
Same as above, I will close this PR as the block is deprecated.